core/src/main/java/org/kiwix/kiwixmobile/core/main/TabsAdapter.java

```diff
@@ -178,12 +186,42 @@ void setTabClickListener(TabClickListener listener) {
     final ImageView content;
     final TextView title;
     final ImageView close;
+    private NightModeConfig nightModeConfig;
+    private final Paint invertedPaint = createInvertedPaint();
 
-    ViewHolder(View v, ImageView content, TextView title, ImageView close) {
+    ViewHolder(View v, ImageView content, TextView title, ImageView close, Context context) {
       super(v);
       this.content = content;
       this.title = title;
       this.close = close;
+
+      SharedPreferenceUtil sharedPreferenceUtil = new SharedPreferenceUtil(context);
```

Comment:
we use dagger to provide dependencies but seeing as how this class isn't created by dagger you will have to pass this in to the constrcutor